terraform-vpc/vpc.tf

```diff
@@ -0,0 +1,116 @@
+data "aws_availability_zones" "iac-azs" {
+    state = "available"
+}
+
+resource "aws_vpc" "da-wordpress-vpc" {
+  cidr_block = "${var.cidr_vpc}"
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+  tags = {
+    Name = "da-wordpress-vpc"
+  }
+}
+
+resource "aws_eip" "da-wordpress-eip" {
+  vpc=true
+}
+
+resource "aws_internet_gateway" "da-wordpress-igw" {
+  vpc_id = "${aws_vpc.da-wordpress-vpc.id}"
+  tags = {
+    Name ="terra-igw"
+  }
+}
+
+resource "aws_nat_gateway" "da-wordpress-nat-a" {
+  allocation_id           = "${aws_eip.da-wordpress-eip.id}"
+  subnet_id               = aws_subnet.public-wp-a.id
+  tags = {
+    Name = "da-wordpress-nat-a"
+  }
+}
+
+resource "aws_nat_gateway" "da-wordpress-nat-b" {
+  allocation_id           = "${aws_eip.da-wordpress-eip.id}"
+  subnet_id               =  aws_subnet.public-wp-b.id
+  tags = {
+    Name = "da-wordpress-nat-b"
+  }
+}
+
```

Comment:
What is this resource be used for?

Reply:
At first,
I was under the impression that we needed a NAT Gateway / Private subnets for our DB.
Or for the NAT Gateway to connect to the private subnet.

Now, 
Because we are going for the Aurora RDS Solution, we just need to connect the WP Instance to the via MySQL port.

So this won't be needed anymore ?

Thoughts

Reply:
Unless you guys decide to put resources that require internet access in a private subnet, there is no need for a NAT GW

Reply:
NAT GW has been removed from VPC